src/views/Cinematic/index.jsx

```
@@ -0,0 +1,243 @@
+import React, { Fragment, useCallback, useEffect, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)